docs/guide/data_migration.md

```diff
@@ -1,19 +1,150 @@
 # Data migration
 
+Migrations are commonly used in Ibexa DXP projects that require the same data to be present across multiple instances.
```

Comment:
You can use migrations in projects that require the same data to be present across multiple instances.
```